helpdesk/templates/helpdesk/ticket_dependency_del.html

```diff
@@ -18,7 +18,7 @@ <h2>Delete Ticket Dependency</h2>
 <p>Are you sure you wish to remove the dependency on this ticket?</p>
 {% endblocktrans %}
 
-<p><a href='../../'><button class="btn btn-primary btn-lg">{% trans "Don't Delete" %}</button></a></p>
+<p><a href='../../../'><button class="btn btn-primary btn-lg">{% trans "Don't Delete" %}</button></a></p>
```

Comment:
Maybe we should use `{{ ticket.get_absolute_url }}` instead of a a relative URL.

Reply:
Changed in 3ea2d1dd501b249c43f1c2ed2ce67f09c6e615d6